src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Overlay/OverlayOperatorTests.cs

```diff
@@ -151,7 +151,7 @@ public void GivenAOCIFileLayer_WhenGenerateDiffOCIFileLayerWithSnapshot_ADiffOCI
             var diffLayers = _overlayOperator.GenerateDiffLayer(fileLayer, baseFileLayer);
 
             File.Delete("TestData/UserFolder/.image/base/defaultlayer.tar.gz");
-            Assert.Equal(819, diffLayers.FileContent.Count());
+            Assert.Equal(820, diffLayers.FileContent.Count());
```

Comment:
Just curious why does the assert value change in this file? It seems we are using the same testing data.

Reply:
Fixed it in the new pr #291